### PR TITLE
BXMSDOC-4516 PAM/DM 7.4.1 release notes

### PR DESCRIPTION
**Pull request by emmurphy1**
See jira for details
[BXMSDOC-4516](https://issues.jboss.org/browse/BXMSDOC-4516)

### CODE REVIEW

File: doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc [hunk 1]
```diff
@@ -188,6 +202,28 @@ Issue: Task operations such as *Start*, *Complete*, or *Claim* are noticeably sl
 
 endif::[]
 
+ifdef::PAM[]
+== {KIE_SERVER}
+.Query-related operations on Process Server running on Red Hat JBoss Web Server or SpringBoot with DB2 return an error [https://issues.jboss.org/browse/RHPAM-2292[RHPAM-2292]]
+
+Issue: Query-related operations on Process Server running on Red Hat JBoss Web Server or SpringBoot with DB2 return an error.
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc]: Query-related operations return errors when Process Server is run on either Red Hat JBoss Web Server or SpringBoot with DB2.

File: doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc [hunk 2]
```diff
@@ -188,6 +202,28 @@ Issue: Task operations such as *Start*, *Complete*, or *Claim* are noticeably sl
 
 endif::[]
 
+ifdef::PAM[]
+== {KIE_SERVER}
+.Query-related operations on Process Server running on Red Hat JBoss Web Server or SpringBoot with DB2 return an error [https://issues.jboss.org/browse/RHPAM-2292[RHPAM-2292]]
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc]: Query-related operations return errors when Process Server is run on either Red Hat JBoss Web Server or SpringBoot with DB2.

**Review by csherrar — COMMENTED** [review]
A few questions and suggestions, otherwise looks good to me.

File: doc-content/enterprise-only/release-notes/rn-741-fixed-issues-ref.adoc [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+[NOTE]
+====
+{PRODUCT_DM} fixed issues apply to {PRODUCT_PAM} as well. For a list of {PRODUCT_DM} 7.4.1 fixed issues, see the https://access.redhat.com/documentation/en-us/red_hat_decision_manager/7.4/html-single/release_notes_for_red_hat_decision_manager_7.4/index[_Release Notes for Red Hat Decision Manager 7.4_]
+====
+
+== {CENTRAL}
+* Guided scorecard rules are not executed through the test scenario [https://issues.jboss.org/browse/RHPAM-2064[RHPAM-2064]]
+* If you configure {KIE_SERVER} and navigate to the *Execution Servers* perspective without deploying a project, an error occurs [https://issues.jboss.org/browse/RHPAM-2031[RHPAM-2031]]
+* An error occurs if you copy a guided rule that has a BigDecimal field [https://issues.jboss.org/browse/RHPAM-2233[RHPAM-2233]]
+//* An exception occurs if you add "The following does not exist:" to the left side of the rule [https://issues.jboss.org/browse/RHPAM-2211[RHPAM-2211]]
```
[the remark this answers is not in the document]
Comment (emmurphy1) [reply doc-content/enterprise-only/release-notes/rn-741-fixed-issues-ref.adoc:16]: Yes. If the jira is not publically visible, I put a note in the jira and comment out until it is made visible (if ever).

**Pull request merged**